_ml-commons-plugin/model-serving-framework.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Model serving framework 
+has_children: false
+nav_order: 110
+---
+
+# Model serving framework
+
+ML commons supports the ability to serve custom models and use those models to make inferences. For those who wish to use train their models using a deep-learning framework, such as Pytorch, TensorFlow, or ONNX, they can upload such models to OpenSearch in order to take advantage of OpenSearch resources, like GPU acceleration.
```

Comment:
We only support deep learning models trained by Pytorch in 2.4.

Reply:
Yes, please remove mention of GPU for now. Let's stick to what we can support